nautobot/core/api/views.py

```diff
@@ -611,13 +613,27 @@ def get(self, request, *args, **kwargs):
 
         # Add additional data so drf-spectacular will use the Token keyword in authorization header.
         response.data["schema_auth_names"] = ["tokenAuth"]
+
+        # Pass `Titles` instance to render title from this view rather than original from Swagger View
+        response.data["view_titles"] = self.view_titles
+        del response.data["title"]
+
         return response
 
 
 class NautobotSpectacularRedocView(APIVersioningGetSchemaURLMixin, SpectacularRedocView):
     """Extend SpectacularRedocView to support Nautobot's ?api_version=<version> query parameter."""
 
     template_name = "redoc_ui.html"
+    view_titles = Titles(titles={"*": "API Documentation"})
+
+    @extend_schema(exclude=True)
+    def get(self, request, *args, **kwargs):
+        response = super().get(request, *args, **kwargs)
+        response.data["view_titles"] = self.view_titles
+        del response.data["title"]
```

Comment:
Same question here - maybe this is what you're alluding to in your comment but if so I'm afraid I don't quite understand.